src/components/shared/page.tsx

```diff
@@ -129,6 +135,25 @@ const getImageMappings = (imageSalt: string, capi: Content): ImageMappings => {
         return { ...blockMappings, ...getContributorMappings(capi.tags, imageSalt) }
 }
 
+const liveblogHydrationProps = (capi: Content): Content => {
+    const { id: contentId, type, webTitle, webUrl, apiUrl, fields, tags, references, isHosted, blocks } = capi;
+    const { main, body, totalBodyBlocks } = blocks as Blocks;
+    const { createdBy, lastModifiedBy, ...mainBlocks } = main as Block;
+
+    return {
+        id: contentId, type, webTitle, webUrl, apiUrl, fields, references, isHosted,
+        tags: tags.map(({ type, webTitle, webUrl, id, references, apiUrl }) => ({ type, webTitle, webUrl, id, references, apiUrl })),
+        blocks: {
+            totalBodyBlocks,
+            main: mainBlocks,
+            body: body?.map(block => {
+                const { createdBy, lastModifiedBy, ...bodyBlocks } = block;
```

Comment:
Do these shadow the `createdBy` and `lastModifiedBy` variables in the outer scope?

Reply:
deduped in a function